src/app/_components/DesktopNavigation.tsx

```diff
@@ -0,0 +1,181 @@
+
+const getInvolvedItems = [
+  {
+    label: PATHS.EVENTS.label,
+    href: PATHS.EVENTS.path,
+    description: 'Overview of upcoming and past events',
+  },
+  {
+    label: PATHS.GRANTS.label,
+    href: PATHS.GRANTS.path,
+    description:
+      'Information on funding opportunities supporting projects that contribute to the growth of the Filecoin Network',
+  },
+  {
+    label: 'Become a Storage Provider',
+    href: 'https://destor.com/destor-network/overview',
+    description: 'Dive right in and become an essential part of the ecosystem',
+  },
+  {
+    label: 'Contribute to the Filecoin Project',
+    href: 'https://docs.filecoin.io/basics/project-and-community/ways-to-contribute',
+    description:
+      'Shape the future of Filecoin by contributing to its code, research, or docs.',
+  },
+]
+
+const communityItems = [
+  {
+    label: PATHS.ECOSYSTEM.label,
+    href: PATHS.ECOSYSTEM.path,
+    description: 'Learn about various ecosystem projects or submit your own',
+  },
+  {
+    label: PATHS.GOVERNANCE.label,
+    href: PATHS.GOVERNANCE.path,
+    description:
+      'Learn about Filecoin Improvement Proposals and other governance mechanisms',
+  },
+]
+
+const learnMoreItem = {
+  label: 'Learn more about Filecoin',
+  href: 'https://docs.filecoin.io/',
+}
+
+type LinkProps = {
+  label: string
+  description: string
+  href: string
+}
+
+function InternalLink({ label, description, href }: LinkProps) {
+  return (
+    <Link
+      href={href as Route}
+      className="inline-block w-full rounded-lg bg-brand-800 p-4 hover:bg-brand-700 focus:outline focus:outline-2 focus:outline-brand-100"
+      aria-label={`${label} page (internal link)`}
+    >
+      <p className="pb-1 font-bold">{label}</p>
+      <p className="text-brand-300">{description}</p>
+    </Link>
+  )
+}
+
+function ExternalLink({ label, description, href }: LinkProps) {
+  return (
+    <a
+      href={href}
+      target="_blank"
+      rel="noopener noreferrer"
+      className="group inline-block w-full rounded-lg border border-brand-500 bg-brand-700 p-4 hover:border-brand-400 focus:border-transparent focus:outline focus:outline-2 focus:outline-brand-100"
+      aria-label={`${label} (opens a new window)`}
+    >
+      <div className="flex gap-1">
+        <p className="pb-2 font-bold">{label}</p>
+        <span className="text-brand-400 group-hover:text-brand-100">
+          <Icon component={ArrowUpRight} size={20} />
+        </span>
+      </div>
+
+      <p className="text-brand-300">{description}</p>
+    </a>
+  )
+}
+
+const getInvolvedInternalItems = getInvolvedItems.filter((item) =>
+  item.href.startsWith('/'),
+)
+
+const getInvolvedExternalItems = getInvolvedItems.filter(
+  (item) => item.href.startsWith('https://') || item.href.startsWith('http://'),
+)
+
+export function DesktopNavigation() {
+  const pathname = usePathname()
+
+  return (
+    <ul
+      className="relative z-10 hidden items-center gap-0.5 lg:flex"
```

Comment:
Could we use just `flex`?

Reply:
But we need `lg:flex` for `hidden` to kick in on smaller screens.

Reply:
Ah, I see! I was thrown off by `items-center gap-0.5` which aren't prefixed.